scripts/primaza.sh

```diff
@@ -49,7 +43,9 @@ pe "mvn clean install -DskipTests -Ppush-images,kubernetes -Dquarkus.container-i
    -Dquarkus.container-image.group=${REGISTRY_GROUP} \
    -Dquarkus.container-image.tag=${IMAGE_VERSION} \
    -Dquarkus.container-image.insecure=true \
-   -Dquarkus.kubernetes.ingress.host=${INGRESS_HOST}"
+   -Dquarkus.kubernetes.ingress.host=${INGRESS_HOST} \
+   -Dgit.sha.commit=$(git rev-parse --short HEAD) \
+   -Dgithub.repo=https://github.com/halkyonio/primaza-poc"
```

Comment:
Is it correct s job fails ? @Sgitario

Reply:
How can you provide default values then ?

Reply:
https://github.com/halkyonio/primaza-poc/pull/121#discussion_r1026196504
and
https://github.com/halkyonio/primaza-poc/pull/121#discussion_r1026196866

Reply:
Thanks. This is what I did ;-)